Makefile

```diff
@@ -82,12 +82,17 @@ ifeq ("$(BUILD_COMPAT)", "1")
 TESTS += ccurl-multi_pow
 endif
 
+TESTS += remotedcurl-new-task
+TESTS += remotedcurl-get-result
```

Comment:
Since the testing of remote interface is not as simple as other testing, I suggest to separate them.
Otherwise, the `make check` would never succeed.

I think we can classify them into two sets.
One is compile the testing and execute them directly, the other is compile but do not execute(which is suitable for remote interface).

Reply:
Per the suggestion of @marktwtn, new test items should not be involved in RabbitMQ broker. You can append the test items which is decoupled with RabbitMQ.